packages/atlas/lib/src/marker_icon.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:meta/meta.dart';
+
+/// Optional Marker icon.
+class MarkerIcon {
+  /// The [assetName] argument must not be null. It should name the main asset
+  /// from the set of images to choose from. File should be of type png
+  final String assetName;
+
+  /// The device pixel ratio where the image will be shown
+  final double devicePixelRatio;
```

Comment:
A few things here:

- I would rename the variable to just `aspectRatio`
- I would update the document to be more specific (also, it does not matter where the image will be shown)
- In the constructor, make it optional and default to 1

Reply:
i just completely removed devicePixelRatio. The developer can just pass in an image of any size